src/js/containers/ImportAccount/ImportByWalletLink.js

```diff
@@ -36,20 +37,48 @@ export default class ImportByWallletLink extends React.Component {
         try {
             var address = await wallet.getAddress()
             this.props.closeParentModal();
-            this.props.dispatch(importNewAccount(address.toLowerCase(),
-                WalletType,
-                null,
-                this.props.ethereum,
-                this.props.tokens, null, null, "Wallet Link"))
-                
+            const chainId = await wallet.getChainId()
```

Comment:
For walletlink, dont need to check chainId

Reply:
I removed the logic for checking chainId